tools/shared.py

```diff
@@ -2557,10 +2586,12 @@ def metadce(js_file, wasm_file, minify_whitespace, debug_info):
     return Building.js_optimizer_no_asmjs(js_file, passes, extra_info=json.dumps(extra_info))
 
   @staticmethod
-  def minify_wasm_imports_and_exports(js_file, wasm_file, minify_whitespace, debug_info):
+  def minify_wasm_imports_and_exports(js_file, wasm_file, minify_whitespace, minify_exports, debug_info):
     logger.debug('minifying wasm imports and exports')
     # run the pass
     cmd = [os.path.join(Building.get_binaryen_bin(), 'wasm-opt'), '--minify-imports-and-exports', wasm_file, '-o', wasm_file]
+    # TODO: Change the above to following when binaryen is updated:
```

Comment:
binaryen has been updated, so I think we can do this now

Reply:
Oh, perfect!